lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -996,7 +997,8 @@ def upload_students_csv(_xmodule_instance_args, _entry_id, course_id, task_input
     task_progress.update_task_state(extra_meta=current_step)
 
     # compute the student features table and format it
-    query_features = task_input.get('features')
+    course = get_course_by_id(course_id, depth=0)
```

Comment:
It would be great to use a `CourseOverview` object here, but looks like a field that is needed, [`is_teams_enabled`](https://github.com/edx/edx-platform/blob/6eec3ec992e9b47751ffcb56e058a887ba2e9108/lms/djangoapps/instructor_analytics/basic.py#L237-L237) isn't available in [`CourseOverviews`](https://github.com/edx/edx-platform/blob/1185fff1ec4e8bbff3747581265a7996b20aba64/openedx/core/djangoapps/content/course_overviews/models.py#L54-L101) for now :/

No action needed. Just a wistful comment. I'll file a ticket to add it.